internal/cmd/kubernetes.go

```diff
@@ -220,7 +223,7 @@ func writeKubeconfig(user *api.User, destinations []api.Destination, grants []ap
 	}
 
 	// move the temp file to overwrite the kube config
-	err = os.Rename(tmpFile.Name(), defaultConfig.ConfigAccess().GetDefaultFilename())
+	err = os.Rename(tmpFile.Name(), configPath)
```

Comment:
Is this right? From the looks of it, it's moving tmpFile into configPath, which _isn't_ kubeconfig. I don't think this needs to change